### ISSUE TITLE
[Accessibility] Fluent Reveal effect should have "base" layer

### ISSUE THREAD

**Issue by Poopooracoocoo** [microsoft/microsoft-ui-xaml#817]
**Describe the bug**

In apps such as Alarms & Clock where there is a wide ListView, or even in NavigationViews, it becomes hard to see what is selected on the opposite side of the reveal effect. 

**Steps to reproduce the bug**

1. Go to Alarms & Clock (with transparency effects enabled)
2. Hover over an alarm on the left edge

**Expected behavior**

The whole item would have a highlight and the light effect would be "on top" of that 

**Version Info**

| Windows 10 version                  | Saw the problem? |
| :--------------------------------- | :-------------------- |
| May 2019 Update (18362)            |  Yes  |
| October 2018 Update (17763)        |  Yes  |

| Device form factor | Saw the problem? |
| :-------------------- | :------------------- |
| Desktop                 |  Yes   |

**Comment by kikisaints** [comment]
Your expected behavior is actually how the Reveal effect works today. However, I think the visibility issue that you're seeing is because that "base" layer's color is too transparent/dim.

For example, today's ListView:
![image](https://user-images.githubusercontent.com/6964369/59051610-2581e680-8842-11e9-9209-19a33ce41fec.png)

Versus one with a "lighter" backplate:
![image](https://user-images.githubusercontent.com/6964369/59052091-22d3c100-8843-11e9-8267-a930d78a3e2d.png)

@kimsealls and @YuliKl for awareness - look into ListViewItem's point over bg being SystemControlHighlightList~~Low~~**Medium**Brush?

**Comment by jevansaks** [comment]
@kikisaints once you've worked this out with design, can you remove the "spec issue" label and add "bug" (and then unassign yourself)?

**Comment by YuliKl** [comment]
@Poopooracoocoo, can you provide a screenshot showing the problem? For both the "today" image that @kikisaints added and the Alarms & Clock app on my screen, I'm not having any difficulty following which item my cursor is currently hovered over.

**Comment by Poopooracoocoo** [comment]
Hmm. I was so sure that Alarms & Clock had that issue. It was either fixed or it was never there in the first place (mandela effect?!). Well, the start menu does.

![image](https://user-images.githubusercontent.com/51363864/59081648-8daff580-8932-11e9-9640-835f39438471.png)

**Comment by kikisaints** [comment]
Start does not add a backplate/"base" on hover - they rely solely on the Reveal to indicate the pointer over state. I can sync with them, but unfortunately Reveal cannot fix that kind of visibility issue.

**Comment by Poopooracoocoo** [comment]
I'm confused. Does that mean each app has to add a backplate on hover?

**Comment by YuliKl** [comment]
UWP apps utilizing Xaml controls don't need to do any extra work to get the default backplates. Windows Shell components like the Start menu are unique and sometimes have custom implementation details.

**Comment by kikisaints** [comment]
Yes, as Yulia stated, the Shell behavior is custom work - they went out of their way to remove the default backplates that you see when you use our controls.

**Comment by Poopooracoocoo** [comment]
Thank you for the information!

**Comment by Poopooracoocoo** [comment]
How would I go about submitting this feedback to the people who work on the Windows shell?

**Comment by kikisaints** [comment]
@Poopooracoocoo perhaps try the Feedback hub? @sravya03 and/or @bschoepke is there any easy way to get accessibility feedback to the shell?

**Comment by bschoepke** [comment]
@Poopooracoocoo please file a Problem in Feedbuck Hub under the Desktop Environment > Start menu category and make sure to include the screenshots. Thanks!

**Comment by Poopooracoocoo** [comment]
https://aka.ms/AA5caio
